### PR TITLE
sdk/js: v0.9.20 release

### PR DESCRIPTION
**Pull request by kev1n-peters**

### CODE REVIEW

File: sdk/js/package-lock.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   "name": "@certusone/wormhole-sdk",
-  "version": "0.9.18",
+  "version": "0.9.19-beta.3",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "@certusone/wormhole-sdk",
-      "version": "0.9.18",
+      "version": "0.9.19-beta.3",
```
Comment (panoel) [inline sdk/js/package-lock.json:9]: Beta?  Does that mean this is going to have to change again relatively soon?

File: sdk/js/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 0.9.19
```
Comment (panoel) [inline sdk/js/CHANGELOG.md:3]: Does this name need to match what is in `package.json`?

File: sdk/js/package-lock.json (hunk 1 above)
Comment (kev1n-peters) [reply sdk/js/package-lock.json:9]: Changed to 0.9.19

**Review by panoel — APPROVED** [review]

**Review by panoel — APPROVED** [review]
Didn't really feel qualified to review the relayer code.

**Review by evan-gray — APPROVED** [review]

**Pull request merged**